### PR TITLE
Added modelbase package

### PR DESCRIPTION
**Pull request by marvinvanaalst**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

File: recipes/modelbase/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+    - modelbase.core
+    - modelbase.ode
+    - modelbase.ode.models
+    - modelbase.ode.simulators
+    - modelbase.ode.utils
+    - modelbase.utils
+
+about:
+  home: "https://gitlab.com/qtb-hhu/modelbase-software"
+  license: GPL-2.0-or-later
```
Comment (dopplershift) [inline recipes/modelbase/meta.yaml:46]:
```suggestion
  license: GPL-3.0-or-later
```

File: recipes/modelbase/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 662682567ecae226718a634ab5162f49d62d262b9e730f58cec09bcdea7f7417
+
+build:
+  number: 0
+  skip: True  # [py2k or py<37]
```
Comment (dopplershift) [inline recipes/modelbase/meta.yaml:14]: This looks like pure Python, so it should be able to be noarch Python:
```suggestion
  number: 0
  noarch: python
```

File: recipes/modelbase/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+  script: {{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv
+
+requirements:
+  host:
+    - python {{ python }}
```
Comment (dopplershift) [inline recipes/modelbase/meta.yaml:19]:
```suggestion
    - python >=3.7
```

File: recipes/modelbase/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+    - pip
+  run:
+    - assimulo
+    - black
+    - dataclasses
+    - matplotlib
+    - {{ pin_compatible('numpy') }}
+    - pandas
+    - python {{ python }}
```
Comment (dopplershift) [inline recipes/modelbase/meta.yaml:28]:
```suggestion
    - python >=3.7
```

File: recipes/modelbase/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+requirements:
+  host:
+    - python {{ python }}
+    - pip
+  run:
+    - assimulo
+    - black
+    - dataclasses
+    - matplotlib
+    - {{ pin_compatible('numpy') }}
```
Comment (dopplershift) [inline recipes/modelbase/meta.yaml:26]: You only need the `pin_compatible` if you're building against NumPy's C-API.
```suggestion
    - numpy
```

File: recipes/modelbase/meta.yaml [hunk 6]
```diff
@@ -0,0 +1,55 @@
...
+
+requirements:
+  host:
+    - python {{ python }}
+    - pip
+  run:
+    - assimulo
+    - black
+    - dataclasses
+    - matplotlib
```
Comment (dopplershift) [inline recipes/modelbase/meta.yaml:25]: Unless you need a PyQt-based GUI, better to only depend on `matplotlib-base`:
```suggestion
    - matplotlib-base
```

**Review by dopplershift — CHANGES_REQUESTED** [review]
Generally in pretty good shape, just mainly a few tweaks to the dependencies.

**Comment by marvinvanaalst** [comment]
Cheers for looking over it so quickly. So far the whole thing has been a very pleasant experience, thanks for the nice work you guys did! :smile:

**Review by dopplershift — APPROVED** [review]

**Pull request merged**